src/test/java/edu/harvard/iq/dataverse/api/DatasetsIT.java

```diff
@@ -1426,6 +1426,13 @@ public void testCreateDeleteDatasetLink() {
         
         // This should fail, because we are attempting to link the dataset 
         // to its own dataverse:
+        Response publishTargetDataverse = UtilIT.publishDataverseViaNativeApi(dataverseAlias,  apiToken);
+        publishTargetDataverse.prettyPrint();
+                        publishTargetDataverse.then().assertThat()
+                        .statusCode(OK.getStatusCode());
```

Comment:
It would be nice to fix the indentation above.

Reply:
will do